src/components/CommentList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CommentOfPost from './CommentOfPost';
+import {comments} from '../comments';
+import './CommentList.css';
+
+class CommentList extends React.Component {
```

Comment:
Use a function instead of class.
you didn't use the potentiality of class
https://ru.reactjs.org/docs/components-and-props.html#function-and-class-components